kedro-datasets/kedro_datasets/polars/__init__.py

```diff
@@ -10,11 +10,14 @@
 CSVDataset: Any
 GenericDataSet: type[GenericDataset]
 GenericDataset: Any
+PolarsDataSet: type[PolarsDataset]
+PolarsDataset: Any
```

Comment:
If this is a new dataset there's no reason to use both the old and new spelling. It should just use the new `Dataset` ending.